packages/connect/src/types/firmware.ts

```diff
@@ -9,25 +12,20 @@ export type FirmwareRange = Record<
     }
 >;
 
-export type IntermediaryVersion = 1 | 2 | 3;
-export const IntermediaryVersion = Type.Union([Type.Literal(1), Type.Literal(2), Type.Literal(3)]);
+export type BinaryInfo = {
+    binary: ArrayBuffer;
+    binaryVersion: VersionArray;
+    releaseVersion?: VersionArray;
+};
 
-export type ReleaseInfo = {
-    changelog: FirmwareRelease[] | null;
-    release: FirmwareRelease;
+export type FirmwareReleaseConfigInfo = {
+    firmwareType: FirmwareType;
+    // TODO: maybe this is not necessary anymore since we have bitcoin-only FW for all models.
```

Comment:
well, if it is always going to be set to true, then it might be removed 🤔 🤷

Reply:
Yes, right now it always true, since all the models have a bitcoin-only version.